packages/typespec-ts/test/integration/routes.spec.ts

```diff
@@ -32,13 +32,94 @@ describe("RoutesClient Rest Client", () => {
     assert.strictEqual(result.status, "204");
   });
 
-  it.skip("should have PathParameters SimpleExpansion Standard primitive", async () => {
+  it("should have allowReserved: true", async () => {
     const result = await client
-      .path("/routes/path/simple/standard/primitive{param}", "a")
+      .path("/routes/path/reserved-expansion/template/{param}", {
+        value: "foo/bar baz",
+        allowReserved: true
+      })
       .get();
     assert.strictEqual(result.status, "204");
   });
 
+  it("should have explode: true array", async () => {
```

Comment:
This case failed with 400 issue and I think we build correct url but failed in server side: https://github.com/Azure/cadl-ranch/issues/731.